packages/cnpj-gen/tests/Specs/CnpjGeneratorOptions.spec.php

```
@@ -0,0 +1,406 @@
+
+                $options->prefix = '11222333';
+
+                expect($options->prefix)->toBe('11222333');
+            });
+        });
+
+        describe('when setting to a nullish value', function () use ($defaultParameters) {
+            it('sets default value for `null`', function () use ($defaultParameters) {
+                $options = new CnpjGeneratorOptions(prefix: '12345');
+
+                $options->prefix = null;
```

PHPStan (2.1.54)

[warning] Property Lacus\BrUtils\Cnpj\CnpjGeneratorOptions::$prefix (string) does not accept null.


(assign.propertyType)